improver/cli/__init__.py

```diff
@@ -207,29 +207,56 @@ def create_constrained_inputcubelist_converter(*constraints):
     These cubes get loaded and returned as a CubeList.
 
     Args:
-        *constraints (str):
-            constraints to be used in the loading of cubes against a cubeList.
+        *constraints (str or list):
+            Constraints to be used in the loading of cubes into a CubeList.
+            If multiple constraints are specified in a list, all provided
+            constraints will be tried as possible options. Only one must
+            identify valid input, otherwise an error will be raised.
```

Comment:
This isn't quite right. This argument is a tuple that may contain strings (one or multiple) or lists (one or multiple). If it contains strings the expectation is that each will return a match (e.g. recursive filter), and if not an error is raised. If the tuple contains lists of constraints, these lists are each treated as a group, with the expectation that each group will return a single match (e.g. nowcast extrapolation), else an error is raised.

I think this doc-string and the raises ValueError doc-string entry below both need to be rewritten to make this clear.

Reply:
I've had a go at re-wording the docstrings as suggested. What do you think?